RELEASE.md

```diff
@@ -2,18 +2,78 @@
 
 This document guides a contributor through creating a release of JupyterLab.
 
+## Automated Releases with the Jupyter Releaser
+
+The recommended way to make a release is to use [`jupyter_releaser`](https://github.com/jupyter-server/jupyter_releaser#checklist-for-adoption).
+
+### Workflow
+
+The full process is documented in https://jupyter-releaser.readthedocs.io/en/latest/get_started/making_first_release.html#making-your-first-release.
+
+Here is a quick summary of the different steps.
+
+#### Draft Changelog
+
+The first step is to generate a new changelog entry for the upcoming release.
+
+We use the "Draft Changelog" workflow as documented here: https://jupyter-releaser.readthedocs.io/en/latest/get_started/making_first_release.html#draft-changelog
+
+The workflow takes a couple of input parameters, for example:
+
+| Input        | Value                 |
+| ------------ | --------------------- |
+| Target       | jupyterlab/jupyterlab |
+| Branch       | master                |
+| Version Spec | next                  |
+| Since        | v4.0.0a15             |
+
+The version spec follows the specification documented bellow in the [Bump Version](#bump-version) section.
+
+We can use `next` when making a `patch` release or a `build` pre-release.
+
+Here is an example screenshot of what it should look like:
+
+![draft-changelog-example](https://user-images.githubusercontent.com/591645/141277239-571c2e6a-e8d5-4af9-aee4-dd52a7dae8ba.png)
```

Comment:
Should this image be included in the repo so the documentation is stand-alone, or perhaps the fields should just be listed out in text instead of a screenshot that will get out of date?

I think instead of a screenshot, a listing of each field and what text is in it is more sustainable as instructions.

Reply:
As mentioned above the releaser docs already have screenshots anyway: https://jupyter-releaser.readthedocs.io/en/latest/get_started/making_first_release.html#making-your-first-release

So we can stick to plain text here.

I find screenshots really useful because we can immediately get the idea of what to do. Agree they can easily get out of date though.